skbio/io/format/embl.py

```diff
@@ -1440,26 +1419,22 @@ def _serialize_feature_table(intervals, indent=21):
 
         # I need to remove two spaces, cause I will add a FT key
         for line in tmp.split("\n"):
-            output += wrapper.wrap(re.sub("^  ", "", line))
+            output += wrapper.wrap(line[2:])
 
         # re add newline between elements, and a final "\n"
         yield "\n".join(output) + "\n"
 
 
 def _parse_date(lines, label_delimiter=None, return_label=False):
-    """Parse embl data records"""
+    """Parse embl date records"""
 
     data = []
-    line = lines[0]
-
-    # take the first line, divide the key from the text
-    label, section = line.split(label_delimiter, 1)
 
-    # add section to data to return
-    data += [section]
+    # take the first line, and derive a label
+    label, section = lines[0].split(label_delimiter, 1)
 
     # read all the others dates and append to data array
-    data.extend(line.split(label_delimiter, 1)[-1] for line in lines[1:])
+    data.extend(line.split(label_delimiter, 1)[-1] for line in lines)
```

Comment:
no need to to do `extend`. you can define `data` here the first time instead of an empty `[]` in line 1431